In russian locale, netcat error messages display as '???????'

Need to set LC_CTYPE along with LC_MESSAGES to correctly display messages in
locales other then C/POSIX
Required for correct i18n support in glibc.

(bug #200875 by Михаил)


--- a/src/netcat.c
+++ b/src/netcat.c
@@ -157,6 +157,7 @@
   connect_sock.domain = PF_INET;
 
 #ifdef ENABLE_NLS
+  setlocale(LC_CTYPE, "");
   setlocale(LC_MESSAGES, "");
   bindtextdomain(PACKAGE, LOCALEDIR);
   textdomain(PACKAGE);
